reframe/frontend/cli.py

```diff
@@ -177,6 +177,9 @@ def main():
         '--module-mappings', action='store', metavar='FILE',
         dest='module_map_file',
         help='Apply module mappings defined in FILE')
+    misc_options.add_argument(
+        '--purge-env', action='store_true', dest='purge_env', default=False,
+        help="Perform a `module purge' before setup")
```

Comment:
It's worth noting a "why" here. "For example for clusters that insist on 'module purge' before any load commands in job scripts."

Reply:
Alright, reworded it a little